docs/source/user_guide/archive_cache.rst

```diff
@@ -0,0 +1,109 @@
+===========================
+Using :mod:`.archive_cache`
+===========================
+
+:mod:`.archive_cache` provides three central fixtures 
+
+- :py:func:`~aiida_testing.archive_cache.create_node_archive` and :py:func:`~aiida_testing.archive_cache.load_node_archive` are used to manage AiiDA archive test files in the pytest environment
```

Comment:
Do we still need these?

Given that they are not documented below, I suggest you either move the mention of them to the end of the document or you remove them (fixtures + documentation) entirely.

Reply:
transformed these into normal functions and moved them into the utils module 👍